central/reports/config/datastore/datastore_impl_postgres_test.go

```diff
@@ -116,6 +128,44 @@ func (s *ReportConfigurationPostgresDatastoreTests) TestMultipleReportNotifiers(
 	s.Equal(reportConfig, foundReportConfig)
 }
 
+func (s *ReportConfigurationPostgresDatastoreTests) TestDeleteCollectionWithReportConfig() {
+
+	reportConfig := fixtures.GetValidReportConfigWithMultipleNotifiersV2()
+	// Test add collection
+	s.addCollectiontoReportConfiguration(reportConfig)
+
+	// Add report config
+	for i, n := range reportConfig.GetNotifiers() {
+		reportConfig.Notifiers[i].Ref = s.storeNotifier(n.GetId())
+	}
+	_, err := s.datastore.AddReportConfiguration(s.ctx, reportConfig)
+	s.Require().NoError(err)
+
+	// Test get
+	err = s.collectionDS.DeleteCollection(s.ctx, reportConfig.ResourceScope.GetCollectionId())
+	s.Require().Error(err)
```

Comment:
What do you want to test here?

Reply:
Test case - when collection is refereed in a report config collectionDS.DeleteCollection should return an error

Reply:
Can you change the comment then? The comment is `Test get` but delete operation is tested. 
`Verify collections referenced in report configuration cannot be deleted`